pkg/kubelet/kubelet_pods.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2016 The Kubernetes Authors.
+Copyright 2016,2019 The Kubernetes Authors.
```

Comment:
I believe the `pull-kubernetes-verify` test is failing because of this change. I'm fairly sure the copyright should remain 2016.

Reply:
I've reverted the change. Thanks!